content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,352 @@
+
+.. image:: Removal/warehouse-settings.png
+   :align: center
+   :alt: :alt: Warehouse settings.
 
 .. note::
-   To work with the *FEFO* strategy, activate the *Expiration Dates* feature.
+   To use the :abbr:`FEFO (First Expiry, First Out)` removal strategy, *expiration dates* need to be
+   activated, as well. To enable this, go to :menuselection:`Inventory app --> Configuration -->
+   Settings`, scroll down to the :guilabel:`Traceability` section, and click the checkbox next to
+   :guilabel:`Expiration Dates`. Remember to click :guilabel:`Save` to save all changes.
 
-Next, you need to define your removal strategy, on *Product Categories* via
-:menuselection:`Inventory --> Configuration --> Product Categories`.
+Now, specific removal strategies can be defined on product categories. To do this, go to
+:menuselection:`Inventory app --> Configuration --> Product Categories`, and choose a product
+category to define the removal strategy on.
 
-.. image:: removal/first-in-first-out.png
+.. image:: Removal/product-category-removal.png
    :align: center
-   :alt: force removal strategy set up as first in first out.
+   :alt: :alt: Removal strategy on a product category.
 
 FIFO (First In, First Out)
 --------------------------
 
-As said, a *FIFO* strategy implies that products stocked first move out first. Companies should use
-that method if they are selling products with short demand cycles, such as clothes, and to ensure
-they are not stuck with outdated styles in stock.
+As explained earlier, the :abbr:`FIFO (First In, First Out)` removal strategy implies that products
+which entered a warehouse's stock first move out first. Companies should use this method if they are
+selling products with short demand cycles, such as clothes, to ensure they are not stuck with
+outdated styles in stock.
 
-For this example, we created three lots of white shirts. Those are from the All/Clothes category,
-where we put *FIFO* as the removal strategy. In our stock location (WH/Stock), we now find the
-three lots available.
+In this example, there are three lots of white shirts. The shirts are from the All/Clothes category,
+where :guilabel:`FIFO` is set as the removal strategy. In the :guilabel:`Inventory Valuation
+Report`, the three different receipts are listed with the amounts.
 
 .. image:: removal/inventory-valuation.png
    :align: center
-   :alt: view of the white shirt lots inventory valuation.
+   :alt: View of the lots of white shirts in the inventory valuation report.
 
-The lot 000001 contains 5 shirts, 000002 contains 3 shirts, and 000003 contains 2. As it can be
-seen above, 000001 has entered the stock first. Let’s create a sale order of six white shirts
-to check that products from that lot are the first ones to move out.
+Lot 000001 contains five shirts, lot 000002 contains three shirts, and lot 000003 contains two
+shirts.
+
+Go to the *Sales* app and click :guilabel:`Create` to create an :abbr:`SO (Sales Order)`. Next,
```

Comment:
```suggestion
Go to the :menuselection:`Sales` app and click :guilabel:`Create` to create an :abbr:`SO (Sales Order)`. Next,
```
I *think* we use menuselection tags when we are telling someone to "go somewhere" on the screen